app/src/main/res/values/strings.xml

```diff
@@ -228,4 +228,5 @@
     <string name="title_undo">Undo</string>
     <string name="title_vendor_delete">Vendor deleted</string>
     <string name="error_load_makers">Sorry, Not load makers successfully !!!</string>
+    <string name="title_searching">Search...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.